### ISSUE TITLE
Incomplete wiki

### ISSUE THREAD

**Issue by officialPlocki** [Panda-Projects/CloudNet-V3-Webinterface#1]
Please include instructions for ...
Nginx,
installation after upload (Setup Website),
installing with composer ("composer install" command),
ssl certificates (or link to a tutorial (e.g. https://pterodactyl.io/tutorials/creating_ssl_certificates .html, suitable for NGINX)
and for creating a user in cloudnet (or changing a password) to make it easier for people without prior knowledge.

**Comment by nicocruw55** [comment]
> Please include instructions for ... Nginx, installation after upload (Setup Website), installing with composer ("composer install" command), ssl certificates (or link to a tutorial (e.g. https://pterodactyl.io/tutorials/creating_ssl_certificates .html, suitable for NGINX) and for creating a user in cloudnet (or changing a password) to make it easier for people without prior knowledge.


Do you know how to create a user and a password? I know you wrote that in June but I would be happy to know

**Comment by officialPlocki** [comment]
Yeah, I know it but many people do use smth. like this for the first time and need more assistance than others.

**Comment by nicocruw55** [comment]
> Yeah, I know it but many people do use smth. like this for the first time and need more assistance than others.


Can you describe what to do? I'm working on it since hours
